bin/cli.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
```

Comment:
```
#!/usr/bin/env bash
```

is best practice, especially for mac since the shipped bash is pretty old.

Reply:
Thanks for the recommendation.